ddtrace/internal/remoteconfig/_connectors.py

```diff
@@ -14,10 +14,9 @@
 
 log = get_logger(__name__)
 
-# Size of the shared variable. It's calculated based on Remote Config Payloads. At 2023-04-26 we measure on stagging
-# RC payloads and the max size of a multiprocess.array was 139.002 (sys.getsizeof(data.value)) and
-# max len 138.969 (len(data.value))
-SHARED_MEMORY_SIZE = 603432
+# Size of the shared variable.
+# It must be large enough to receive at least 2500 IPs or 2500 users to block.
+SHARED_MEMORY_SIZE = 0x100000
```

Comment:
NB: Dumb question, how did you calculate this?

Reply:
I took the smallest power of two that was making the python tracer pass the system tests for large payloads.
It's an increase from previous value that was not working, but I hope that it's a reasonable one (from 0.6 MB to 1MB of memory).